### PR TITLE
created page that allows to add question

### PR DESCRIPTION
**Pull request by jygh98**
This issue was dependent on the OAuth issue #34 , now we reinstated an add question feature back to survey admin. This will allow us to directly add questions to the database from the website instead of having to manually add it from psql.

*NOTE: do PR for issue #34  FIRST before accepting this.

### CODE REVIEW

File: ubuntuStation/forms.py [hunk 1]
```diff
@@ -77,3 +89,11 @@ class LoginForm(FlaskForm):
     username=TextField('username', description=u'Username', validators=[DataRequired()])
     password=PasswordField('password', description=u'Password', validators=[DataRequired()])
 
+class UserForm(FlaskForm):
```
Comment (carlosparadis) [inline ubuntuStation/forms.py:92]: Wasn't this added on https://github.com/erdl/survey_admin/pull/34/files#diff-c56dbff25916fe369f8214c588886a95 ?

I suspect this will lead to a merge conflict. Also will need to be re-named to `CreateUser` if will stay here ,as requested on the other PR, for code readability.

File: ubuntuStation/static/bootstrap.css [hunk 2]
```diff
@@ -182,8 +182,8 @@ legend {
   padding: 0;
 }
 textarea {
-  height: 100px;
-  width: 300px:
+  height: 120px;
```
Comment (carlosparadis) [inline ubuntuStation/static/bootstrap.css:185]: What is this change about? Seems unrelated.

File: ubuntuStation/views.py [hunk 3]
```diff
@@ -289,3 +324,28 @@ def show_questions():
     #entries=[dict(questiontext=q.questiontext, questionurl=q.questionurl) for q in question]
     entries=question
     return render_template('show_questions.html', questions=entries)
+
+@app.route('/createuser', methods=['GET', 'POST'])
+@login_required
+def create_user():
```
Comment (carlosparadis) [inline ubuntuStation/views.py:330]: I am a bit confused on what is this createuser form and how it is different from the current UserForm class / page.

**Review by carlosparadis — CHANGES_REQUESTED** [review]
I need help understanding some of the changes. Please see below. **Also, there is something wrong with this commit**. There are a number of changes, including the Google login logo, that seems entirely unrelated to the issue of `adding questions`. The 15 files changes also appear to be way too high for such a feature.

**I am led to believe this commit suffers from a similar issue we went over before, where the branching was done on top of another local branch, instead of the one from origin**.

**Comment by ryantanaka** [comment]
ryan do this
